cpp/src/slice2java/msbuild/slice2java.vcxproj

```diff
@@ -115,6 +115,7 @@
   </ItemGroup>
   <ItemGroup>
     <ClInclude Include="..\Gen.h" />
+    <ClInclude Include="..\GenCompat.h" />
```

Comment:
The build wan't failing without these?

Reply:
No, that is just to display the headers in Visual Studio Solution Explorer. The compiler still finds them because they are in out of the includes search paths.